src/main/java/thePackmaster/cards/darksoulspack/PrisonersChain.java

```diff
@@ -0,0 +1,28 @@
+package thePackmaster.cards.darksoulspack;
+
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import thePackmaster.cards.AbstractPackmasterCard;
+import thePackmaster.powers.darksoulspack.PrisonersChainPower;
+import thePackmaster.util.Wiz;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+import static thePackmaster.util.Wiz.*;
+
+public class PrisonersChain extends AbstractDarkSoulsCard {
+    public final static String ID = makeID("PrisonersChain");
+    // intellij stuff power, self, uncommon, , , , , 3, 2
+
+    public PrisonersChain() {
+        super(ID, 1, CardType.POWER, CardRarity.UNCOMMON, CardTarget.SELF);
+        baseMagicNumber = magicNumber = 3;
```

Comment:
Hmm, also something to keep an eye on. This seems a bit weak, but it could widly scale out of control quickly with your attack for each debuff on your attack.

Personally I'd probably go with 2(3) temp strength whenever a debuff is applied to you, but that would take away the chance to stack it for the super payoff. Interesting conondrum, we'll just have to wait and see how it plays out.

Reply:
wouldn't be against this change, but obviously temp strength typically applies a debuff as well (the str down at end of turn).

Reply:
Oh very good point, I had forgotten.